spec/requests/omniauth_facebook_authorize_spec.rb

```diff
@@ -65,23 +66,34 @@
   end
 
   context 'with valid login info for an existing user with same uid/provider' do
-    let!(:user) { create(:user, provider: provider, uid: uid) }
-    before { omniauth_authenticate(valid_oauth_login) }
+    let!(:user) { create(:user) }
+    let!(:identity) { create(:identity, provider: provider, uid: uid, user: user)}
 
-    it 'DOES NOT create a new user' do
-      expect(response).to be_redirect
-      expect { follow_redirect! }.not_to change(User, :count)
+    before(:each) do
+      omniauth_authenticate(valid_oauth_login)
     end
 
-    context 'following redirect' do
+    it 'redirects to callback' do
+      expect(response).to redirect_to user_facebook_omniauth_callback_path
```

Comment:
@krnjn this is failing:

I've experimented with setting the path explicitly using `ENV.fetch('SITE_HOST')` and `ENV.fetch('SITE_PROTOCOL')` to explicitly specify the path. But there must be something simpler that I'm missing, since the test directly above successfully tests `         expect(response).to redirect_to edit_user_registration_path(setting: 'onboard')
`.

```
       Expected response to be a redirect to <http://www.example.com/users/auth/facebook/callback> but was a redirect to <http://localhost:3000/users/auth/facebook/callback>.
       Expected "http://www.example.com/users/auth/facebook/callback" to be === "http://localhost:3000/users/auth/facebook/callback".
     # ./spec/requests/omniauth_facebook_authorize_spec.rb:77:in `block (3 levels) in <top (required)>'
```